### ISSUE TITLE
Style/MixinGrouping triggers inappropriately on RSpec "include" usage

### ISSUE THREAD

**Issue by radar** [rubocop-hq/rubocop#5799]
*Be clear, concise and precise in your description of the problem.
Open an issue with a descriptive title and a summary in grammatically correct,
complete sentences.*

*Use the template below when reporting bugs. Please, make sure that
you're running the latest stable RuboCop and that the problem you're reporting
hasn't been reported (and potentially fixed) already.*

*Before filing the ticket you should replace all text above the horizontal
rule with your own words.*

--------

## Expected behavior

No offenses detected.

## Actual behavior

```

spec/models/props/employees/basic_props_spec.rb:14:37: C: Style/MixinGrouping: Put include mixins in separate statements.
  it { expect(subject[:locales]).to include { { code: :en, label: 'English (US English)' } } }
                                    ^^^^^^^

1 file inspected, 1 offense detected
```

## Steps to reproduce the problem

Create an RSpec file, put this code in it:

```
describe SomeClass do
  it { expect(subject[:locales]).to include { { code: :en, label: 'English (US English)' } } }
end
```

Run `rubocop` over this file. It should report the offense.

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.55.0 (using Parser 2.5.1.0, running on ruby 2.4.2 x86_64-darwin17)
```

**Comment by Drenmi** [comment]
Definitely a false positive. 😅 Will have a look at the cop.
